reference/v8.json

```diff
@@ -1439,6 +1475,33 @@
         }
       }
     },
+    "fill-extrude-height": {
+      "type": "number",
+      "function": "interpolated",
+      "zoom-function": true,
+      "property-function": true,
+      "default": 0,
+      "minimum": 0,
+      "units": "meters",
+      "doc": "The height with which to extrude this fill layer.",
+      "transition": true
+    },
+    "fill-extrude-base": {
+      "type": "number",
+      "function": "interpolated",
+      "zoom-function": true,
+      "property-function": true,
+      "default": 0,
+      "minimum": 0,
+      "units": "meters",
+      "doc": "The height with which to extrude the base of this layer.",
+      "transition": true,
+      "requires": [
+        {
+          "<=": "fill-extrude-height"
```

Comment:
This `<=` key to a `requires` object is new, right? So its introduction will require changes to documentation generation code and Studio.